bundles/org.openhab.binding.icloud/src/main/resources/OH-INF/i18n/iCloud_de.properties

```diff
@@ -10,6 +10,8 @@ icloud.account-thing.parameter.apple-id.label=Apple-ID
 icloud.account-thing.parameter.apple-id.description=Apple-ID (E-Mail Adresse), um Zugriff zur iCloud zu erhalten.
 icloud.account-thing.parameter.password.label=Passwort
 icloud.account-thing.parameter.password.description=Passwort der Apple-ID, um Zugriff zur iCloud zu erhalten.
+icloud.account-thing.parameter.code.label=Code
+icloud.account-thing.parameter.code.description=Code für die 2-Faktor-Authentifizierung des Accounts.
```

Comment:
Please remove these changes, translations should be updated through Crowdin. For reference, see https://www.openhab.org/docs/developer/utils/i18n.html#managing-translations